go.mod

```diff
@@ -65,10 +65,10 @@ require (
 	github.com/gocql/gocql v1.5.2
 	github.com/golang/mock v1.6.0
 	github.com/google/uuid v1.6.0
-	github.com/googleapis/gax-go/v2 v2.12.0
+	github.com/googleapis/gax-go/v2 v2.12.2
 	github.com/gorilla/mux v1.8.1
 	github.com/grandcat/zeroconf v1.0.0
-	github.com/hamba/avro/v2 v2.15.0
+	github.com/hamba/avro/v2 v2.21.1
```

Comment:
According to your table, the vulnerability was in < 2.13.0. We were using 2.15.0 so should have been safe. No need to update this dependency then: in a release branch, it's best to be as conservative as possible.